docs/develop/go/versioning.mdx

```diff
@@ -212,80 +204,24 @@ w := worker.New(c, "your_task_queue_name", workerOptions)
 ```
 
 That's all you need to do in your Worker code.
-Importantly, if you start this Worker, it won't receive any tasks.
-That's because you need to tell the Task Queue about your Worker's Build ID first.
-
-### Tell the Task Queue about your Worker's Build ID
-
-Now you can use the SDK (or the Temporal CLI) to tell the Task Queue about your Worker's Build ID.
-You might want to do this as part of your CI deployment process.
-
-```go
-// ...
-err := client.UpdateWorkerBuildIdCompatibility(ctx, &client.UpdateWorkerBuildIdCompatibilityOptions{
-   TaskQueue: "your_task_queue_name",
-   Operation: &client.BuildIDOpAddNewIDInNewDefaultSet{
-      BuildID: "deadbeef",
-   },
-})
-```
-
-This code adds the `deadbeef` Build ID to the Task Queue as the sole version in a new version set, which becomes the default for the queue.
-New Workflows execute on Workers with this Build ID, and existing ones will continue to process by appropriately compatible Workers.
-
-If, instead, you want to add the Build ID to an existing compatible set, you can do this:
-
-```go
-// ...
-err := client.UpdateWorkerBuildIdCompatibility(ctx, &client.UpdateWorkerBuildIdCompatibilityOptions{
-   TaskQueue: "your_task_queue_name",
-   Operation: &client.BuildIDOpAddNewCompatibleVersion{
-      BuildID:                   "deadbeef",
-      ExistingCompatibleBuildId: "some-existing-build-id",
-   },
-})
-```
-
-This code adds `deadbeef` to the existing compatible set containing `some-existing-build-id` and marks it as the new default Build ID for that set.
-
-You can also promote an existing Build ID in a set to be the default for that set:
-
-```go
-// ...
-err := client.UpdateWorkerBuildIdCompatibility(ctx, &client.UpdateWorkerBuildIdCompatibilityOptions{
-   TaskQueue: "your_task_queue_name",
-   Operation: &client.BuildIDPromoteIDWithinSet{
-      BuildID: "some-existing-build-id",
-   },
-})
-```
+Importantly, if you start this Worker, it won't receive any tasks until you set up assignment rules.
+See the [Pre-release README.md](https://github.com/temporalio/temporal/blob/main/docs/worker-versioning.md) for more information.
 
-You can also promote an entire set to become the default set for the queue. New Workflows will start using that set's default.
+### Specify versions for Activities, Child Workflows, and Continue-as-New Workflows
 
-```go
-// ...
-err := client.UpdateWorkerBuildIdCompatibility(ctx, &client.UpdateWorkerBuildIdCompatibilityOptions{
-   TaskQueue: "your_task_queue_name",
-   Operation: &client.BuildIDPromoteSet{
-      BuildID: "some-existing-build-id",
-   },
-})
-```
-
-### Specify versions for Commands
-
-By default, Activities, Child Workflows, and Continue-as-New use the same compatible version set as the Workflow that invoked them if they're also using the same Task Queue.
+By default, Activities, Child Workflows, and Continue-as-New Workflows stay on the build of the workflow that spawned them if they are also being run on the same task queue.
```

Comment:
Workflow and Task Queue belong to the magic "must capitalize" words.

"stay on the build", I _think_ you mean "retain" or "inherit" the Build ID.
"if they are also being run on" -> "when run on". Simpler, clearer.

nit: "spawned them" is correct, but I think you might consider "created" "initiated", etc.